### ISSUE TITLE
Hysteresis -> OffsetOff logic?

### ISSUE THREAD

**Issue by avollkopf** [Manuel83/craftbeerpi4#21]
Should this not be 'above' instead of 'below'? -> This would only work if the OffsetOff would be negative which might be confusing to the user.

`  Property.Number(label="OffsetOff", configurable=True, description="Offset below target temp when heater should switched off")`

Currently, the parameter offset_off would indeed need to be negative that the elif part would work.
```

            while True:
                sensor_value = self.get_sensor_value(self.kettle.get("sensor")).get("value")
                target_temp = self.get_kettle_target_temp("oHxKz3z5RjbsxfSz6KUgov")
                if sensor_value < target_temp - self.offset_on:
                    await self.actor_on(self.heater)
                elif sensor_value >= target_temp - self.offset_off:
                    await self.actor_off(self.heater)
                await asyncio.sleep(1)
```

Proposal would be to change the description to 'above' and convert the elif part. In the code above, the kettle_target_temp is also still hardcoded. This has been changed in the code below.

```
            while True:
                sensor_value = self.get_sensor_value(self.kettle.get("sensor")).get("value")
                target_temp = self.get_kettle_target_temp(self.kettle.get("id"))
                if sensor_value < target_temp - self.offset_on:
                    await self.actor_on(self.heater)
                elif sensor_value >= target_temp + self.offset_off:
                    await self.actor_off(self.heater)
                await asyncio.sleep(1)
```

**Comment by Manuel83** [comment]
fixed with version 4.0.0.22

**Comment by avollkopf** [comment]
@Manuel83 . I looked into cbpi3 and I was obviously wrong. Both parameters have to be 'below' and the sign in he equation has to be both times a '-'. I mixed this up with a fermenter logic for heater and cooler. Sorry.

There is also an issue in the current version of the hysteresis plugin right above the while loop:

`self.get_actor_state()`

This is causing an error since no actor is mentioned and probably not required anyhow at this position.
